test/dutchExchange-Tulip.js

```diff
@@ -42,14 +43,29 @@ const setupContracts = async () => {
   } = contracts)
 }
 
-const getBalance = async (acct, token) => ((await dx.balances.call(token.address, acct)).toNumber())
+// I know, it's gross
+// add wei converter
+/* eslint no-extend-native: 0 */
+Number.prototype.toWei = function toWei() {
+  return (this * 10 ** 18)
+}
+Number.prototype.toEth = function toEth() {
+  return (this / 10 ** 18)
+}
```

Comment:
this still leaves big number errors in cases:
```
> (200000000000000001).toEth()
0.2

> (4.1).toWei()
4099999999999999500
```
As long as you stick to `40|50` and the like numbers it's all good, but maybe consider `web3.fromWei/toWei`